src/Tools/dotnet-counters/KnownData.cs

```diff
@@ -31,21 +31,21 @@ private static IEnumerable<CounterProvider> CreateKnownProviders(string runtimeV
                     new CounterProfile{ Name="cpu-usage", Description="Percentage of time the process has utilized the CPU (%)", SupportedVersions=new[] { net30, net31, net50 } },
                     new CounterProfile{ Name="working-set", Description="Amount of working set used by the process (MB)", SupportedVersions=new[] { net30, net31, net50 } },
                     new CounterProfile{ Name="gc-heap-size", Description="Total heap size reported by the GC (MB)", SupportedVersions=new[] { net30, net31, net50 } },
-                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs / min", SupportedVersions=new[] { net30, net31, net50 } },
-                    new CounterProfile{ Name="gen-1-gc-count", Description="Number of Gen 1 GCs / min", SupportedVersions=new[] { net30, net31, net50 } },
-                    new CounterProfile{ Name="gen-2-gc-count", Description="Number of Gen 2 GCs / min", SupportedVersions=new[] { net30, net31, net50 } },
+                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs between interval", SupportedVersions=new[] { net30, net31, net50 } },
```

Comment:
```suggestion
                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs between intervals", SupportedVersions=new[] { net30, net31, net50 } },
```

or 

```suggestion
                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs per interval", SupportedVersions=new[] { net30, net31, net50 } },
```

or 

```suggestion
                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs since last interval", SupportedVersions=new[] { net30, net31, net50 } },
```

(I'm not sure if they are averages or flat counts, so choose accordingly.